app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
What is the story on this new `ApplicationRecord` abstract class?  This isn't a pattern I've seen before, although our one Rails 6 application does not use ActiveRecord, so we might not have encountered it.

Reply:
This is the new default starting from Rails 5, see: https://github.com/rails/rails/pull/22567